### PR TITLE
Update deadline for submissions

### LINKED ISSUE #20: Miranda Mundt submitted a request due 2024-03-18
**Request type**: 

- [x] Website change / addition
- [x] Email to conference listserv
- [x] Email to all US-RSE members
- [x] Social media blast (LinkedIn, Mastodon, X/Twitter)
- [x] Slack

**Email** mmundt@sandia.gov

**Purpose** Update on new deadline for submissions; remind that the deadline is coming!!

**Content**  https://docs.google.com/document/d/1Wh21NBWzoOPhbldn0VIER9eHu3w7h_MnF1hqWRoL9ls/edit 

- [x] I've informed the requestor that this is complete

### PR DESCRIPTION
**Pull request by jsubida**
## Description

Change deadline from March 22 to April 1

Closes #20 

## Checklist:

- [ ] (Committee Chairs Only) I have posted the link for the PR in the usrse slack (#usrse-2024-conference-planning-committee) to ask for content reviewers
- [ ] I have previewed changes locally
- [ ] I have updated the README.md if necessary

cc **ADD GITHUB HANDLES HERE**

### CODE REVIEW

File: pages/about/dates.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ set_last_modified: true
 ## Submissions
 
 - **Submissions open**: Monday, February 12 for all but notebooks. Notebooks to open Monday, February 26.
-- **BoFs, Workshops, and Tutorials**: Due Friday, March 22. Notifications Friday, April 26.
+- **BoFs, Workshops, and Tutorials**: Due Monday, April 1. Notifications Friday, April 26.
```
Comment (mrmundt) [inline pages/about/dates.md:13]: Shoot, please also push the notifications date to May 3. I forgot these were on the pages - my bad!

File: pages/participate/participate.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ Accepted abstracts will be placed on the website ahead of the conference.
 ## Important Dates
 
 - **Submissions open**: Monday, February 12 for all but notebooks. Notebooks to open Monday, February 26.
-- **BoFs, Workshops, and Tutorials**: Due Friday, March 22. Notifications Friday, April 26.
+- **BoFs, Workshops, and Tutorials**: Due Monday, April 1. Notifications Friday, April 26.
```
Comment (mrmundt) [inline pages/participate/participate.md:45]: Shoot, please also push the notifications date to May 3. I forgot these were on the pages - my bad!

**Comment by jsubida** [comment]
@mrmundt All good! I got to experience pull request comments in VSCode for the first time!

File: pages/about/dates.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ set_last_modified: true
 ## Submissions
 
 - **Submissions open**: Monday, February 12 for all but notebooks. Notebooks to open Monday, February 26.
-- **BoFs, Workshops, and Tutorials**: Due Friday, March 22. Notifications Friday, April 26.
+- **BoFs, Workshops, and Tutorials**: Due Monday, April 1. Notifications Friday, May 1.
```
Comment (mrmundt) [inline pages/about/dates.md:13]: May 3rd!

File: pages/participate/participate.md [hunk 4]
```diff
@@ -42,7 +42,7 @@ Accepted abstracts will be placed on the website ahead of the conference.
 ## Important Dates
 
 - **Submissions open**: Monday, February 12 for all but notebooks. Notebooks to open Monday, February 26.
-- **BoFs, Workshops, and Tutorials**: Due Friday, March 22. Notifications Friday, April 26.
+- **BoFs, Workshops, and Tutorials**: Due Monday, April 1. Notifications Friday, May 1.
```
Comment (mrmundt) [inline pages/participate/participate.md:45]: May 3rd!

**Review by mrmundt — APPROVED** [review]

**Comment by jsubida** [comment]
Ope. Done!

**Pull request merged**